src/components/Article.jsx

```diff
@@ -17,27 +17,17 @@ function Article() {
     });
   }, []);
 
-  const handleClickVoteUp = () => {
-    patchArticle(article_id, 1)
-      .then((updatedArticle) => {
-        setArticleVotes((currentArticleVotes) => {
-          return currentArticleVotes + 1;
-        });
-      })
-      .catch((err) => {
-        setApiError(err.response);
-      });
-  };
+  const handleClick = (event) => {
+    setArticleVotes((currentArticleVotes) => {
+      return currentArticleVotes + event;
+    });
```

Comment:
Is event an appropriate name here? What is event? What kind of data does it hold?